examples/rust/bzsstack/src/main.rs

```diff
@@ -0,0 +1,95 @@
+use std::time::Duration;
+
+use std::u64;
+
+use std::io::Error;
+use std::thread;
+use std::result::Result;
+
+extern crate libbpf_rs;
+use libbpf_rs::MapFlags;
+
+extern crate nix;
+use nix::unistd::getpid;
+
+#[path = "bpf/.output/bzsstack.skel.rs"]
+mod bzsstack;
+use bzsstack::*;
+
+extern crate blazesym;
+use blazesym::*;
+
+fn create_symbol_files_cfg() -> Result<Vec<SymbolFileCfg>, Error> {
+    let cfgs = vec![SymbolFileCfg::Kernel { kallsyms: None, kernel_image: None },
+		    SymbolFileCfg::Process { pid: Some(0) }];
+
+    Ok(cfgs)
+}
+
+fn main() -> Result<(), Error> {
+    let cfgs = create_symbol_files_cfg()?;
+    let resolver = BlazeSymbolizer::new()?;
+
+    let skel_builder = BzsstackSkelBuilder::default();
+
+    let mut open_skel = skel_builder.open().unwrap();
+    let pid = getpid().as_raw();
+    open_skel.bss().my_pid = pid;
+    let mut skel = open_skel.load().unwrap();
+
+    let _handle_tp =
+	skel.progs_mut().handle_tp().attach().unwrap();
```

Comment:
is this how cargo fmt format this? If you haven't please run `cargo fmt` so that everything is formatted consistently

Reply:
Roger1